tests/scale/virt-api.go

```diff
@@ -25,29 +29,21 @@ var _ = Describe("[sig-compute] virt-api scaling", decorators.SigCompute, func()
 	numberOfNodes := 0
 
 	setccs := func(ccs v12.CustomizeComponents) (oldcss v12.CustomizeComponents) {
-		originalKv := util.GetCurrentKv(virtClient)
-		kv, err := virtClient.KubeVirt(originalKv.Namespace).Get(context.Background(), originalKv.Name, v1.GetOptions{})
-		Expect(err).ToNot(HaveOccurred())
+		kv := util.GetCurrentKv(virtClient)
 		oldcss = kv.Spec.CustomizeComponents
-		kv.Spec.CustomizeComponents = ccs
-		EventuallyWithOffset(1, func() error {
```

Comment:
```
Replace update with patch at virt-api tests
Signed-off-by: Daniel Sionov <dsionov@redhat.com>
```

I know why `Update` + retry is a bad practice in most cases, but it may not be clear to all future reviewers. Would you add an explanation why this commit makes things better?

Reply:
Sure, added explanation to the commit